sandbox/grist/runtests.py

```diff
@@ -9,11 +9,14 @@
   ./sandbox/nacl/bin/run python /grist/runtests.py [--xunit]
 """
 import codecs
+import logging
 import os
 import sys
 import unittest
 sys.path.append('/thirdparty')
 
+logging.basicConfig(level=logging.DEBUG)
```

Comment:
Being more verbose when running tests. This commit may be isolated, and may move in a dedicated PR.